services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -1244,10 +1247,10 @@ public CallInfo stopCall(String callId, boolean remove) throws CallNotFoundExcep
 
         if (remove) {
           // Log metrics - call deleted
-          LOG.info(metricMessage(userId, call, OPERATION_CALL_DELETED, STATUS_OK, System.currentTimeMillis() - opStart, null));
+          LOG.info(metricMessage(userId, call, OPERATION_CALL_DELETED, STATUS_OK, System.currentTimeMillis() - opStart, null, 0.0D, 0));
```

Comment:
can you pass null if the attribute is not relevant ?

Reply:
ok ,I will change